libraries/AC_AttitudeControl/AC_AttitudeControl.cpp

```diff
@@ -246,11 +246,10 @@ void AC_AttitudeControl::input_quaternion(Quaternion& attitude_desired_quat, Vec
         _ang_vel_target.z = input_shaping_angle(wrap_PI(attitude_error_angle.z), _input_tc, get_accel_yaw_max_radss(), _ang_vel_target.z, ang_vel_target.z, radians(_ang_vel_yaw_max), _dt);
     } else {
         _attitude_target = attitude_desired_quat;
+        _attitude_target.to_euler(_euler_angle_target);
```

Comment:
Why are you adding this here?

Reply:
`_euler_angle_target` used in `ang_vel_to_euler_rate` on 255 before its updated in `attitude_controller_run_quat`

Reply:
Yep, that makes sense.